api-auto-registration/usage.hbs.md

```diff
@@ -114,4 +114,14 @@ spec:
         kind: HTTPProxy
         name: my-httpproxy
 
-```
\ No newline at end of file
+```
+
+## <a id='cors'></a>CORS
```

Comment:
Maybe a little more descriptive title would read better? Setting up CORS for OpenAPI specs

Reply:
Changed